src/ProjectOrigin.WalletSystem.Server/Repositories/CertificateRepository.cs

```diff
@@ -76,8 +76,8 @@ public async Task<IEnumerable<CertificateViewModel>> GetAllOwnedCertificates(str
                     JOIN Slices s ON de.Id = s.DepositEndpointId
                     JOIN Certificates c ON s.CertificateId = c.Id
                     LEFT JOIN Attributes a ON c.Id = a.CertificateId AND c.RegistryId = a.RegistryId
-                    JOIN Registries r ON c.RegistryId = r.Id
-                    WHERE w.Owner = @owner";
+                    JOIN Registries r ON c.RegistryId = r.Id 
+                    WHERE w.Owner = @owner AND s.SliceState = 1";
```

Comment:
What is SliceState = 1?

Reply:
It means that it is available. SliceState is an enum value:

```cs
public enum SliceState
{
    Available = 1,
    Slicing = 2, // Reserved
    Registering = 3,
    Sliced = 4,
    Transferred = 5,
}
```

Reply:
Please see the changes in 21e120f5463f4b312f8ede728dd24bc0294cdb5f. That should be easier to read.